contrail-ui/tslint.json

```diff
@@ -5,6 +5,8 @@
         "tslint-react"
     ],
     "jsRules": {},
-    "rules": {},
+    "rules": {
+        "object-literal-sort-keys": false
```

Comment:
this is illegal

Reply:
Redux standard is that we type field at the top and this rule causes an error for it